### PR TITLE
httpx-sse 0.4.0

### PR DESCRIPTION
**Pull request by SC426**
**Destination channel:** defaults

### Links

- [PKG-6659](https://anaconda.atlassian.net/browse/PKG-6659)
- [Upstream repository](https://github.com/florimondmanca/httpx-sse/tree/0.4.0)
- [Upstream changelog/diff](https://github.com/florimondmanca/httpx-sse/blob/0.4.0/CHANGELOG.md)
- Relevant dependency PRs:
  - `httpx-sse` ➡️ `langchain-community`

### Explanation of changes:

- Fork from conda forge
- Recipe standardization
- Linter fixes
- Skip s390x

[PKG-6659]: https://anaconda.atlassian.net/browse/PKG-6659?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by JamesRobertsonGames — COMMENTED** [review]
Approved, assuming the draft PR isn't intentional in this instance but let me know if you need another review

**Review by JamesRobertsonGames — APPROVED** [review]
Approved, assuming the draft PR isn't intentional in this instance but let me know if you need another review

**Review by cbouss — APPROVED** [review]

**Comment by SC426** [comment]
@JamesRobertsonGames Thank you for reviewing! I'm trying to make it a habit to draft instead of create a PR from the get go, but I always forget to make the update before posting to the review channel.

**Pull request merged**
